cluster/aws.go

```diff
@@ -446,6 +480,25 @@ func (c *AWSCluster) UpdateCluster(request *components.UpdateClusterRequest) err
 	kubicornCluster.ServerPools[1].MaxCount = updateCluster.Amazon.NodeMaxCount
 
 	log.Debug("Get reconciler")
+
+	// Set up credentials TODO simplify
+	runtimeParam := cutil.RuntimeParameters{
+		AwsProfile: "",
+	}
+	clusterSecret, err := GetSecret(c)
+	if err != nil {
+		return err
+	}
+	if clusterSecret.SecretType != secret.Amazon {
+		return errors.Errorf("missmatch secret type %s versus %s", clusterSecret.SecretType, secret.Amazon)
+	}
+	awsCred := credentials.NewStaticCredentials(
+		clusterSecret.Values["AWS_ACCESS_KEY_ID"],
```

Comment:
`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` should be made a const as its used in multiple places.